src/sections/SMI/smi.style.js

```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+
+const SMIWrapper = styled.section`
+    .description{
+        margin: 6.25rem -0.94rem 7.5rem -0.94rem;
+        h6{
+            font-size: 20px;
+            color: #1E2117;
+            font-weight: semi-bold;
+            // ${props => props.theme.textColor};
+        }
+        h1{
+            color: black;
```

Comment:
Redundant

Reply:
h1 is given dark jungle green, but the design demands black.

Reply:
None of the designs have the header assigned the color jungle green. So, that will not be an issue.

Reply:
Moreover, it won't look bad and much different if we use jungle green....but the good part will be that we will adhere to the brand colors.